### PR TITLE
SD-540: add option to control logging for deployments

### PR DESCRIPTION
**Pull request by domagojlatecki**

### CODE REVIEW

File: util/helpers.ts [hunk 1]
```diff
@@ -7,12 +7,18 @@ const config = {
   confirmationsForDeploy: 1
 }
 
-export async function deployStablecoin(deployer: Signer, supply: number | string, precision: number | string, confirmations: number = config.confirmationsForDeploy): Promise<Contract> {
+function log(message: string, opts?: { logOutput: boolean }) {
+  if (!opts || opts?.logOutput === true) {
+    console.log(message);
+  }
+}
```
Comment (githinho) [inline util/helpers.ts:14]: This function is defined in two files: this one and `campaign-deployer.ts`. Maybe extract it to `utils/utils.ts`.

**Review by githinho — COMMENTED** [review]
The file `package-lock.json` should be included in git.

**Comment by domagojlatecki** [comment]
> The file `package-lock.json` should be included in git.

It had no changes to commit.

**Comment by githinho** [comment]
> > The file `package-lock.json` should be included in git.
> 
> It had no changes to commit.

You are right! I just saw removed lines from that file.

**Pull request merged**